docs/content/commands/rclone_purge.md

```diff
@@ -7,6 +7,8 @@ description: "Remove the path and all of its contents."
 
 Remove the path and all of its contents.
 
+The concurrency of this operation is controlled by the `--transfers` global flag.
```

Comment:
I just checked the source code - it is actually `--checkers` which controls the concurrency here.

Also note that the backend may implement this command directly in which case `--checkers` is ignored.